src/Sylius/Bundle/AdminBundle/Resources/config/app/config.yml

```diff
@@ -1,4 +1,5 @@
 imports:
+    - { resource: "@SyliusUiBundle/Resources/config/app/config.yml" }
     - { resource: 'twig_hooks/**/*.yaml' }
```

Comment:
This hurts my eyes, can we in-line it with the rest? (prefix w/ `@SyliusAdminBundle` like the ones below)